lib/dolly/document.rb

```diff
@@ -132,5 +144,9 @@ def init_doc options
       self.doc['_id'] = self.class.namespace( normalized_id ) if normalized_id
     end
 
+    def valid_properties(options)
+      options.keys.any?{ |option| _properties.map(&:name).include?(option.to_s) }
```

Comment:
Will this allow only updating existing properties on a document? or will this save even new added properties on update?